pkg/clients/clients.go

```diff
@@ -212,6 +217,14 @@ func SetScheme(crScheme *runtime.Scheme) error {
 		return err
 	}
 
+	if err := nmstatev1.AddToScheme(crScheme); err != nil {
```

Comment:
According to commit message it's not related to your PR

Reply:
I didnt make any changes to this file. I believe I incorrectly created a commit including the previous PR. Need to remove.